### PR TITLE
Update documentation of  SUMMARY keyword

### LINKED ISSUE #3823: Update the documentation related to SUMMARY keyword
The refcase is not plotted anymore and we need to add that the history vector needs to be explicitly (or with wildcat) declared.

### PR DESCRIPTION
**Pull request by hnformentin**
**Issue**
Resolves #3823

### CODE REVIEW

File: docs/reference/configuration/data_types.rst [hunk 1]
```diff
@@ -589,20 +589,20 @@ multiple vectors:
...
 ::
 
    REFCASE eclipse/refcase/CASE
-   SUMMARY WWCT:* WGOR:*
+   SUMMARY WWCT*:* WGOR*:*
    SUMMARY F*
    SUMMARY G*:NORTH
 
-will load the ``WWCT`` and ``WGOR`` vectors for all wells, all field related
+will load the ``WWCT`` and ``WWCTH`` (``WGOR`` and ``WGORH``) vectors for all wells, all field related
```
Comment (berland) [inline docs/reference/configuration/data_types.rst:605]: Why `WGOR` in parantheses? (and you may line wrap the paragraph after the edits)

Comment (hnformentin) [reply docs/reference/configuration/data_types.rst:605]: Nice suggestion. Thanks.

**Review by berland — APPROVED** [review]
Nice corrections! Remember to squash the commits.

**Comment by hnformentin** [comment]
Please test this!

**Comment by hnformentin** [comment]
Jenkins test this please

**Comment by xjules** [comment]
Jenkins test this please

**Pull request merged**